go/libkb/id_table.go

```diff
@@ -1206,9 +1218,6 @@ func (idt *IdentityTable) proofRemoteCheck(hasPreviousTrack, forceRemoteCheck bo
 		}
 	}
 
-	var pc ProofChecker
-	pc, res.err = NewProofChecker(p)
-
 	if res.err != nil {
```

Comment:
I think we need to move this check up. We don't want to clobber a real proof error with a Tor warning, right?